src/main/kotlin/no/nav/syfo/consumer/narmesteleder/NarmesteLederConsumer.kt

```diff
@@ -113,15 +129,27 @@ class NarmesteLederConsumer @Autowired constructor(
         return HttpEntity<Any>(headers)
     }
 
-    private fun getAnsatteUrl(aktorId: String): String {
-        return "$syfonarmestelederBaseUrl/narmesteLeder/$aktorId"
+    private fun entityWithOboToken(innbyggerIdent: String): HttpEntity<*> {
```

Comment:
Kan vurdere å skille ut det som er felles med entity-metoden over i en egen funksjon (det som lager headers for en gitt token)

Reply:
Ja, jeg vurderte det, men det er en ekstra header for det nye kallet, så da tenkte jeg at det blir litt rart uansett 🤔 (Også er tanken at den opprinnelige entity-metoden skal bort snart 🤞

Reply:
Ok, da er det greit 👍